src/pages/Register.js

```diff
@@ -0,0 +1,55 @@
+import { render } from '@testing-library/react';
+import React from 'react'
+import register from '../assets/styles/pages/register.css'
+import { FaUserAlt, FaEnvelope, FaKey } from "react-icons/fa";
+import { useState } from 'react'
+import FormTutor from '../components/FormTutor';
+
+function Register() {
+  const [form, setForm] = useState({ role: 'student' })
+  const [errorData, setErrorData] = useState('')
+
+  function addData(e) {
+    setForm({ ...form, [e.target.name]: e.target.value })
```

Comment:
When you need to use previous state, add callback function.` setForm((form) => {...form, [e.target.name]: e.target.value }`

Reply:
Done